src/main/scala/KotlinCompile.scala

```diff
@@ -79,7 +79,7 @@ object KotlinReflection {
     val servicesClass = cl.loadClass("org.jetbrains.kotlin.config.Services")
     val messageCollectorClass = cl.loadClass("org.jetbrains.kotlin.cli.common.messages.MessageCollector")
     val commonCompilerArgsClass = cl.loadClass("org.jetbrains.kotlin.cli.common.arguments.CommonCompilerArguments")
-    val argsClass = cl.loadClass("org.jetbrains.kotlin.relocated.com.sampullara.cli.Args")
+    val argsClass = cl.loadClass("org.jetbrains.kotlin.com.sampullara.cli.Args")
```

Comment:
Is it always true that only the latest version will be used? If not, perform the lookup then fallback if it fails